<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserTableForm</class>
 <widget class="QDialog" name="UserTableForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>S R B | Usuarios</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../public/img/icon-SRB.png</normaloff>../../../public/img/icon-SRB.png</iconset>
  </property>
  <property name="styleSheet">
   <string>background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>860</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Usuarios</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAgregarUsuario">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>69</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Verdana&quot;;
background-color: rgb(75, 207, 255);
color: rgb(255, 255, 255);
border-radius: 10px;
padding: 10px;</string>
   </property>
   <property name="text">
    <string>Agregar Usuario</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableUsers">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>860</width>
     <height>350</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>7</number>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nombre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Apellido</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Email</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Usuario</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Rol</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Acciones</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnCerrar">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>490</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Verdana&quot;;
background-color: rgb(127, 0, 0);
padding: 10px;
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string>Volver al menu principal</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_image_3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../public/img/logo-SRB.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
